uavcan/equipment/ice/reciprocating/1120.Status.uavcan

```diff
@@ -0,0 +1,199 @@
+#
+uint30 FLAG_FUEL_PRESSURE_SUPPORTED             = 256
+uint30 FLAG_FUEL_PRESSURE_BELOW_NOMINAL         = 512     # Under-pressure warning
+uint30 FLAG_FUEL_PRESSURE_ABOVE_NOMINAL         = 1024    # Over-pressure warning
+#
+# Detonation warning. This flag is optional.
+# This warning is cleared immediately after broadcasting is done if detonation is no longer happening.
+#
+uint30 FLAG_DETONATION_SUPPORTED                = 2048
+uint30 FLAG_DETONATION_OBSERVED                 = 4096    # Detonation condition observed warning
+#
+# Misfire warning. This flag is optional.
+# This warning is cleared immediately after broadcasting is done if misfire is no longer happening.
+#
+uint30 FLAG_MISFIRE_SUPPORTED                   = 8192
+uint30 FLAG_MISFIRE_OBSERVED                    = 16384   # Misfire condition observed warning
+#
+# Oil pressure. These flags are optional; either none of them or all of them are supported.
+#
+uint30 FLAG_OIL_PRESSURE_SUPPORTED              = 32768
+uint30 FLAG_OIL_PRESSURE_BELOW_NOMINAL          = 65536   # Under-pressure warning
+uint30 FLAG_OIL_PRESSURE_ABOVE_NOMINAL          = 131072  # Over-pressure warning
+#
+# Debris warning. This flag is optional.
+#
+uint30 FLAG_DEBRIS_SUPPORTED                    = 262144
+uint30 FLAG_DEBRIS_DETECTED                     = 524288  # Detection of debris warning
+
+#
+# Reserved space
+#
+void16
+
+#
+# Engine load estimate.
+# Unit: percent.
+# Range: [0, 127].
+#
+uint7 engine_load_percent
+
+#
+# Engine speed.
+# Unit: revolutions per minute.
+#
+uint17 engine_speed_rpm
+
+#
+# Spark dwell time.
+# Unit: millisecond.
+#
+float16 spark_dwell_time_ms
+
+#
+# Atmospheric (barometric) pressure.
+# Unit: kilopascal.
+#
+float16 atmospheric_pressure_kpa
+
+#
+# Engine intake manifold pressure.
+# Unit: kilopascal.
+#
+float16 intake_manifold_pressure_kpa
+
+#
+# Engine intake manifold temperature.
+# Unit: kelvin.
+#
+float16 intake_manifold_temperature
+
+#
+# Engine coolant temperature.
+# Unit: kelvin.
+#
+float16 coolant_temperature
+
+#
+# Oil pressure.
+# Unit: kilopascal.
+#
+float16 oil_pressure
+
+#
+# Oil temperature.
+# Unit: kelvin.
+#
+float16 oil_temperature
+
+#
+# Fuel pressure.
+# Unit: kilopascal.
+#
+float16 fuel_pressure
+
+#
+# Instant fuel consumption estimate.
+# The estimated value should be low-pass filtered in order to prevent aliasing effects.
+# Unit: (centimeter^3)/minute.
+#
+float32 fuel_consumption_rate_cm3pm
+
+#
+# Estimate of the consumed fuel since the start of the engine.
+# This variable MUST be reset when the engine is stopped.
+# Unit: centimeter^3.
+#
+float32 estimated_consumed_fuel_volume_cm3
+
+#
+# Throttle position.
+# Unit: percent.
+#
+uint7 throttle_position_percent
+
+#
+# The index of the publishing ECU.
+#
+uint6 ecu_index
+
+#
+# Spark plug activity report.
+# Can be used during pre-flight tests of the spark subsystem.
+#
+uint3 spark_plug_usage
+#
```

Comment:
@EShamaev i'm not very experienced with engines, could you mind explaining to me what "spark plug usage" is meant to indicate? My best guess is that this is to differentiate between single and dual ignition systems. I suppose it's not for indicating which spark plugs _per cylinder_ are active?